src/use-swr.ts

```diff
@@ -72,7 +37,7 @@ const broadcastState: Broadcaster = (
   isValidating,
   shouldRevalidate = false
 ) => {
-  const [, , CACHE_REVALIDATORS] = getGlobalState(cache)
+  const [, , CACHE_REVALIDATORS] = SWRGlobalState.get(cache)!
```

Comment:
maybe use type gaurd or just disable this lint rule?

Reply:
Adding some type guards would be great. 👍